### PR TITLE
Mirador "zen mode"

### LINKED ISSUE #72: Simplify MS view in Mirador 2
- Hide annotations
- Hide thumbnails

(More like "Zen" mode)

### PR DESCRIPTION
**Pull request by simonwiles**
closes #72

### CODE REVIEW

File: js/components/DashTabs.js [hunk 1]
```diff
@@ -155,7 +155,18 @@ class DashTabs extends React.Component {
...
+          sidePanel: false,
+          displayLayout: false,
+          bottomPanel: false,
+          canvasControls: {
+            annotations: {
+              annotationLayer: false,
+              annotationCreation: false,
+            },
+            imageManipulation: {
+              manipulationLayer: false,
```
Comment (csbailey5t) [inline js/components/DashTabs.js]: Should we leave the manipulation layer on? I could maybe see some cases where someone might want to adjust brightness or contrast to try to see something more effectively

Comment (simonwiles) [reply js/components/DashTabs.js]: I'm happy either way -- I had thought the preference was towards removing as much as possible (à la 'zen mode').  Happy for you to take that call if you have a strong feeling for or against.

Comment (csbailey5t) [reply js/components/DashTabs.js]: Sounds good. MP hasn't ever mentioned those tools, but I think it's good to leave them in for now, mention them to him, and then we can remove if he wants them gone.

Comment (simonwiles) [reply js/components/DashTabs.js]: Done -- thanks!

**Pull request merged**